Phaser/phaser-workshop.adoc

```diff
@@ -130,12 +134,15 @@ function spelUpdaten() {
     }
 
     if (cursors.left.isDown) {
+        // Maak de horizontale snelheid van de speler 160 naar links
         speler.setVelocityX(-160);
         speler.anims.play('naar_links', true);
     } else if (cursors.right.isDown) {
+        // Maak de horizontale snelheid van de speler 160 naar rechts
         speler.setVelocityX(160);
         speler.anims.play('naar_rechts', true);
     } else {
+        // Maak de horizontale snelheid van de speler 0 (staat dus stil)
```

Comment:
(of springt recht omhoog/valt recht naar beneden - dit is alleen de _horizontale_ snelheid. Het poppetje staat alleen stil als zowel de horizontale als de verticale snelheid nul zijn.